jplag/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java

```diff
@@ -113,29 +118,21 @@ private static List<String> getComparisonNames(List<JPlagComparison> comparisons
         return names;
     }
 
-    // Currently, only one metric can be obtained.
-
     /**
-     * Gets the used metric in a JPlag comparison.
+     * Gets the used metrics in a JPlag comparison. As Max Metric is included in every JPlag run, this always include Max
+     * Metric.
      * @return A list contains Metric DTOs.
      */
     private static List<Metric> getMetrics(JPlagResult result) {
         List<Metric> metrics = new ArrayList<>();
-        metrics.add(new Metric(result.getOptions().getSimilarityMetric().name(), result.getOptions().getSimilarityThreshold(),
-                intArrayToList(result.getSimilarityDistribution()), getTopComparisons(result.getComparisons())));
-        return metrics;
-    }
 
-    /**
-     * Converts JPlagComparison to a DTO for displaying only comparisons. See
-     * {@link #generateComparisonReports(JPlagResult)} for a more detailed representation of a comparison.
-     * @return List containing TopComparison DTOs.
-     */
-    private static List<TopComparison> getTopComparisons(List<JPlagComparison> comparisons) {
-        List<TopComparison> topComparisons = new ArrayList<>();
-        comparisons.forEach(comparison -> topComparisons.add(
-                new TopComparison(comparison.getFirstSubmission().getName(), comparison.getSecondSubmission().getName(), comparison.similarity())));
-        return topComparisons;
+        Metric averageMetric = metricMapper.getAverageMetric(result);
+        metrics.add(averageMetric);
+
+        Metric maxMetric = metricMapper.getMaxMetric(result);
+        metrics.add(maxMetric);
+
+        return metrics;
```

Comment:
`return List.of(metricMapper.getAverageMetric(result), metricMapper.getMaxMetric(result));`